packages/connect-query/package.json

```diff
@@ -22,16 +21,13 @@
   "type": "module",
   "sideEffects": false,
   "main": "./dist/cjs/index.js",
+  "module": "./dist/esm/index.js",
   "exports": {
     ".": {
-      "node": {
-        "import": "./dist/proxy/index.js",
-        "require": "./dist/cjs/index.js"
-      },
-      "module": "./dist/esm/index.js",
-      "import": "./dist/proxy/index.js",
+      "import": "./dist/esm/index.js",
       "require": "./dist/cjs/index.js"
-    }
+    },
+    "./package.json": "./package.json"
```

Comment:
Why export package.json?

Reply:
It's a good callout, I was mostly looking to make things more consistent with react-query, which also exports "./package.json" but I couldn't find the reasoning. I also was reading this [extensive investigation](https://blog.isquaredsoftware.com/2023/08/esm-modernization-lessons/#problems-with-next-js-and-react-server-components) which also had that export (again, with no explanation I could find).

Reply:
You typically export package.json because it's otherwise inaccessible to consumers respecting the exports fields. Unless we have a concrete reason to provide package.json, we should not provide it.